tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1084,10 +1084,14 @@ def pad_to_bounding_box_internal(image, offset_height, offset_width,
   Args:
     image: 4-D Tensor of shape `[batch, height, width, channels]` or 3-D Tensor
       of shape `[height, width, channels]`.
-    offset_height: Number of rows of zeros to add on top.
-    offset_width: Number of columns of zeros to add on the left.
-    target_height: Height of output image.
-    target_width: Width of output image.
+    offset_height: Number of rows of zeros to add on top.Must be 1-D `Tensor` 
+      of dtype int32 or int64.Can also a python integer.
```

Comment:
Missing space before "Can".

Reply:
Are you sure it's a 1-D tensor and not a 0-D tensor (i.e. a scalar)?  Here and throughout.